python_wrapper/susi_python/models.py

```diff
@@ -96,11 +96,14 @@ def __init__(self, columns):
 
 
 class MapAction(BaseAction):
-    def __init__(self, latitude, longitude, zoom):
+    def __init__(self, latitude, longitude, zoom= None):
```

Comment:
Check Python coding style (PEP 8).

Reply:
updated @hongquan

Reply:
Still not correct: https://www.python.org/dev/peps/pep-0008/#other-recommendations